Uplift "Cortex XDR by Palo Alto Networks" Pack to Golden Standard. (#1668)
demisto/content-docs: content-repo/extra-docs/packs/palo-alto-networks-cortex-xdr---investigation-and-response.md

---
id: palo-alto-networks-cortex-xdr
title: Cortex XDR by Palo Alto Networks
description: Automates Cortex XDR incident response, and includes custom Cortex XDR incident views and layouts to aid analyst investigations.
---
The Cortex XDR by Palo Alto Networks pack automates Cortex XDR incident response, and includes custom Cortex XDR incident views and layouts to aid analyst investigations.

Cortex XDR is a detection and response app that natively integrates network, endpoint, and cloud data to stop sophisticated attacks.

Effectively responding to these attacks requires security teams to consolidate data from multiple sources. Critical time is wasted switching between tools and performing repetitive tasks, allowing the attack to escalate further.

> This pack can provide significant assistance.

Reviewer: ```suggestion
This pack can provide significant assistance in responding to attacks.
```

## What does the pack do?

- **Automated Incident Synchronization**: Keeps incidents consistently updated between Cortex XDR and Cortex XSOAR, maintaining data integrity and reducing manual effort.
- **Comprehensive Alert Management**: Automates alert handling, including enrichment, severity assessment, and remediation, for a streamlined incident response process.
- **Advanced Threat Hunting**: Leverages the XQL Query Engine for in-depth data analysis, enabling proactive threat hunting and investigation.

> - **Ready-to-Use Playbooks**: a range of pre-built playbooks optimized for common security workflows, allowing teams to quickly deploy effective responses with minimal configuration.

Reviewer: ```suggestion
- **Ready-to-Use Playbooks**: A range of pre-built playbooks optimized for common security workflows, allowing teams to quickly deploy effective responses with minimal configuration.
```

The playbooks included in this pack can help you save time and keep your incidents and indicators in sync with Cortex XDR. They also help automate repetitive tasks:

- Syncs and updates Cortex XDR incidents.
- Triggers a sub-playbook to handle each alert by type.
- Extracts and enriches all relevant indicators from the source alert.
- Hunts for related IOCs.
- Assesses the severity of the incident.
- Collaborates with the analyst to determine the appropriate remediation path or classify the incident as a false positive, based on gathered data and incident severity.
- Remediates the incident by blocking malicious indicators and isolating compromised endpoints.

> ## The Palo Alto Networks Cortex XDR - Investigation and Response pack enables the following flows:

Reviewer: What is the correct name of the pack? 
Is it The Cortex XDR by Palo Alto Networks pack  or The Palo Alto Networks Cortex XDR - Investigation and Response pack?  Make sure the correct name is used everywhere.

- [Lite Incident Handling](#lite-incident-handling) - A lite playbook for handling Palo Alto Networks Cortex XDR incidents, which encompasses incident enrichment, investigation, and response for each incident.
- [Device Control Violations](#device-control-violations) - Fetch device control violations from XDR and communicate with the user to determine the reason the device was connected.
- [XDR Incident Handling](#xdr-incident-handling) - Compare incidents in Palo Alto Networks Cortex XDR and Cortex XSOAR, and update the incidents appropriately.

> - [Cloud IAM User Access Investigation](#cloud-iam-user-access-investigation) - Investigates and responds to Cortex XDR Cloud alerts where an Cloud IAM user's access key is used suspiciously to access the cloud environment.

Reviewer: ```suggestion
- [Cloud IAM User Access Investigation](#cloud-iam-user-access-investigation) - Investigates and responds to Cortex XDR Cloud alerts where a Cloud IAM user's access key is used suspiciously to access the cloud environment.
```

> - [Cortex XDR - Cloud Cryptomining](#Cortex-XDR-Cloud-Cryptomining) - Investigates and responds to Cortex XDR XCloud

Reviewer: should this be "Investigates and responds to Cortex XDR Cloud" or "Investigates and responds to Cortex XDR XCloud"

- **XDR Risky Users**
- **XDR Similar Incidents**
- **XDR Starred**
- **XDR Status v2**
- **XDR URL**
- **XDR User Count**
- **XDR Users**

### Indicator Fields

- XDR status: The indicator status in XDR.
- Fields from the **Common Types** pack.

### Indicator Types

- Types from the **Common Types** pack (such as File, Email, Domain, URL and more).

### Integrations

#### Cortex XDR - IOC

Allows to manage Indicators of Compromise (IOCs) seamlessly within Cortex XDR from Cortex XSOAR. This integration enables security teams to add, update, and remove IOCs efficiently, streamlining threat intelligence workflows and bolstering incident response capabilities.

#### Palo Alto Networks Cortex XDR - Investigation and Response

Allows security teams to automate and streamline incident response workflows by interacting directly with Cortex XDR. With this integration, users can investigate, respond to, and manage incidents efficiently within the Cortex XSOAR platform.
Key capabilities include retrieving incidents, isolating endpoints, executing remediation actions, and fetching forensic data to enhance incident investigation and resolution.

#### Cortex XDR - XQL Query Engine

> Enables to execute XQL queries on your data sources within Cortex XSOAR, facilitating advanced threat hunting and data analysis.

Reviewer: ```suggestion
Enables executing XQL queries on your data sources within Cortex XSOAR, facilitating advanced threat hunting and data analysis.
```

After the remediation, if there are no new alerts, the playbook stops the alert sync and closes the XDR incident and investigation. For performing the bidirectional sync, the playbook uses the incoming and outgoing mirroring feature added in XSOAR version 6.0.0. After the Calculate Severity - Generic v2 sub-playbook’s run, Cortex XSOAR will be treated as the single source of truth for the severity field, and it will sync only from Cortex XSOAR to XDR, so manual changes for the severity field in XDR will not update in the XSOAR incident.

#### [Cortex XDR Incident Sync](https://xsoar.pan.dev/docs/reference/playbooks/cortex-xdr-incident-sync)

> Deprecated. Do not use this playbook when enabling the incident mirroring feature added in XSOAR version 6.0.0. Compares incidents in Palo Alto Networks Cortex XDR and Cortex XSOAR, and updates the incidents appropriately. When an incident is updated in Cortex XSOAR, the XDRSyncScript will update the incident in XDR. When an incident is updated in XDR, the XDRSyncScript will update the incident fields in Cortex XSOAR and rerun the current playbook.

Reviewer: ```suggestion
Deprecated. Do not use this playbook when enabling the incident mirroring feature added in Cortex XSOAR version 6.0.0. Compares incidents in Palo Alto Networks Cortex XDR and Cortex XSOAR, and updates the incidents appropriately. When an incident is updated in Cortex XSOAR, the XDRSyncScript will update the incident in Cortex XDR. When an incident is updated in Cortex XDR, the XDRSyncScript will update the incident fields in Cortex XSOAR and rerun the current playbook.
```

2. Define the inputs for the [Cortex XDR disconnected endpoints](https://xsoar.pan.dev/docs/reference/playbooks/cortex-xdr-disconnected-endpoints) playbook.

   Note: The scheduled run time and the timestamp playbook input must be identical. If the job recurs every 7 days, the timestamp should be 7 days as well.

> ### Create A XDR API Key and retrieve url

Reviewer: ```suggestion
### Create XDR API Key and retrieve URL
```
